security-release-process.md

```diff
@@ -44,30 +45,34 @@ The Fix Lead will give the Fix Team access to a private security repository to d
 The Fix Lead will remove the Fix Team from the private security repository. 
 
 ### Retrospective
 
 These steps should be completed after the Release Date. The retrospective process [should be blameless](https://landing.google.com/sre/book/chapters/postmortem-culture.html).
 
-The Fix Lead will send a retrospective of the process to the Gardener mailing list [gardener@googlegroups.com](mailto:gardener@googlegroups.com) including details on everyone involved, the time line of the process, links to relevant PRs that introduced the issue, if relevant, and any critiques of the response and release process.
-The Release Managers and Fix Team are also encouraged to send their own feedback on the process to  the Gardener mailing list [gardener@googlegroups.com](mailto:gardener@googlegroups.com). Honest critique is the only way we are going to get good at this as a community.
+The Fix Lead will send a retrospective of the process to the [Gardener mailing list](#communication-channel) including details on everyone involved, the time line of the process, links to relevant PRs that introduced the issue, if relevant, and any critiques of the response and release process.
+The Release Managers and Fix Team are also encouraged to send their own feedback on the process to the [Gardener mailing list](#communication-channel). Honest critique is the only way we are going to get good at this as a community.
+
+
+### Communication Channel
+
+The [private](#private-disclosure-process) or [public disclosure process](#public-disclosure-process) should be triggered exclusively by writing an e-mail to [secure@sap.com](mailto:secure@sap.com).
+
+Gardener security announcements will be communicated by the Fix Lead sending an e-mail to the [Gardener mailing list](https://groups.google.com/forum/#!forum/gardener) (reachable via [gardener@googlegroups.com](mailto:gardener@googlegroups.com)) as well as posting a link in the [Gardener Slack channel](https://kubernetes.slack.com/messages/CB57N0BFG/details/). Public discussions about Gardener security announcements and retrospectives, will primarily happen in the Gardener mailing list. Thus Gardener community members who are interested in participating in discussions related to the Gardener Security Release Process are encouraged to join the Gardener mailing list ([how to find and join a group](https://support.google.com/groups/answer/1067205?hl=en))
+
+The members of the [Gardener Security Team](#gardener-security-team) are subscribed to the private [Gardener Security mailing list](https://groups.google.com/forum/#!forum/gardener-security) (reachable via [gardener-security@googlegroups.com](mailto:gardener@googlegroups.com)). 

```

Comment:
The `mailto:` link seems to be wrong. it should be `mailto:gardener-security@googlegroups.com` rather than `mailto:gardener@googlegroups.com`